__tests__/server.test.js

```diff
@@ -127,4 +127,45 @@ describe('GET /api/articles', () => {
             expect(errorMsg).toBe('404: Not Found')
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments for given article_id', () => {
```

Comment:
We are expecting the comments to be served in a way that shows the most recent comments first - are we testing for that?